trieste/acquisition/function/function.py

```diff
@@ -762,3 +763,74 @@ def acquisition(x: TensorType) -> TensorType:
         return tf.exp(tf.linalg.logdet(covariance + jitter))
 
     return acquisition
+
+
+class BALD(SingleModelAcquisitionBuilder):
```

Comment:
we should avoid abbreviations if possible, perhaps `LearningByDisagreement`?

Reply:
wouldn't `ActiveLearningByDisagreement` be more suitable? since Trieste also has a long-named acquisition function like `BatchMonteCarloExpectedImprovement`

Reply:
that's fine as well